---
title: "The *Two-Wing* Pattern of Microbial Communities under Arbitrary Environmental Disturbances"
author: "Yushi Tang"
date: "`r Sys.Date()`"
toc: true
number-sections: false
highlight-style: pygments
format:
  html: 
    code-fold: true
    html-math-method: katex
editor_options: 
  chunk_output_type: console
bibliography: references.bib
---

```{r library}
library(ggplot2)       # for generating plots
library(latex2exp)     # for plot text latex
library(cowplot)       # for merging plots
library(gridExtra)     # for griding plots
library(grid)          # for griding plots
library(matrixStats)   # for matrix computations
```

## Abstract

Here we describe a *Two-Wing* pattern of microbial communities that we have observed in both natural environments and experimental settings.

## Introduction

*Why is it extremely difficult to analyze the structure of microbial communities under arbitrary environmental disturbances?*

-   **A complicated mixture of multiple subgroups.**

-   **Extremely large community size.**

-   **A considerable amount of zero counts.**

-   **The issue of small sample size.**

-   **Observable environmental factors may not be sufficient enough to categorize sample sites and to characterize microbes' living environments.**

-   **The difficulty of detecting functional microbes in the whole community and understanding their ecological roles.**

-   **Lacking a general probabilistic method for microbial communities in both natural environment and experimental settings.**

## Material and Methods

### A natural costal study area and sample collection

Ten sampling sites in Hangzhou Bay and twelve sampling sites in two nearby effluent receiving areas (ERAs) were chosen (@fig-mean_var A). In April 2018, a total of 66 surface sediment samples (triplicate samples in each site) were collected using grab sampler (Van Veen, Hydro-Bios, Germany), and stored at -80 °C after transportation to the laboratory. A part of sediment sample should be freeze-dried for DNA extraction. According to the manufacturer's instructions, total DNA was extracted from 0.25g lyophilized sediments using DNeasy® PowerSoil Kit (QIAGEN, USA). The integrity, concentration, and purity of DNA were evaluated following our previous study [@su2020].

### An experimental anoxic/oxic (AO) system and sample collection

A total of 33 samples were processed by the AO bio-reactor. These samples were collected at 11 factories (triplicate samples in each site) from dyeing (5 sites), medicine (4 sites), and pesticides (2 sites) industry through four procedures in the AO system including the stages of *Influent*, *Anoxic*, *Oxic*, and *Effluent*. `Describe DNA extraction.`

### 16S rRNA gene sequencing

The primer pair 515F (5'- GTG CCA GCM GCC GCG GTA A-3') and 806R (5'- GGA CTA CHV GGG TWT CTA AT-3') were used to amplify the V4 region of bacterial 16S rRNA gene from each purified DNA sample. The PCR amplification program and generation of amplicon sequencing libraries followed the method in our previous study [@dai2020].

### Sequencing data processing

The 16S rRNA gene amplicon sequences were processed using the pipeline of QIIME2 (v2020.11) [@bolyen2019]. All raw sequences were first filtered using DADA2 algorithm with default parameters to generate high-quality amplicon sequence variants (ASVs) [@callahan2016]. A total of 51,441 ASVs were taxonomically assigned against the SILVA SSU 138 database with a $99\%$ sequence identity threshold. To equalize the impact of sequence number variation across different samples, we rarefied all samples based on the lowest sequencing depth (114, 325 sequences) for further analysis.

### Taxanomy

In total, each sample replicate contains 44,525 counts of 29,536 OTUs for AO data and 114,325 counts of 51,441 OTUs for Hangzhou data.

### Environmental factors

-   AO data:

    -   $\text{N.NO}_2$, $\text{N.NH}_4$, $\text{TP}$, $\text{COD}$, $\text{pH}$, $\text{DO}$, $\text{Salinity}$;

-   Hangzhou data:

    -   $\text{Temp.}$, $\text{pH}$, $\text{Salinity}$, $\text{TN}$, $\text{TP}$, $\text{TOC}$, $\text{Oil}$, $\text{Cr}$, $\text{As}$, $\text{Cu}$, $\text{Zn}$, $\text{Cd}$, $\text{Pb}$, $\text{Hg}$, $\text{MPs}$ (for bay).

### Zero-inflation criteria

For both AO and Hangzhou data, we have collected three replicates frome each sample sites. If over two thirds of these replicates have zero counts at either one of these sample sites, we would identify the corresponding microbial abundance as a zero-inflated record. This detection was completed by the function `is_zero_infla` in our R package `PMCosm`.

### Over-dispersion test

This statistical test was implemented by the function `dispersiontest` in R package `AER` and wrapped up by the function `is_overdispersion` in our R package `PMCosm`.

### Fitting the Gamma-Poisson model (GPM)

@fisher1943 first proposed using GPM to characterize the abundance distribution of the underlying population based on the observed random samples. For microbes that do not have inflated zero counts while having significant over-dispersion pattern ($p$-value $<0.05$), we fitted GPM using the well-established function `glm.nb` in the R package `MASS` (version 7.3-57). For microbes that have inflated zero counts and significant over-dispersion pattern, we fitted a zero-inflated Gamma-Poisson model using another classical function `zeroinfl` in the R package `pscl` (version 1.5.5). For microbes that do not have significant over-dispersion pattern, we fitted a standard Poisson model using the base R function `glm`.

### Dimensionality reduction methods

-   principal component analysis (PCA)
-   principal coordinates analysis (PCoA)
-   canonical correspondence analysis (CCA)

### Diversity analysis

-   $\beta$-diversity analysis [@koleff2003]

## Results

### The mean-variance relationship suggests the over-dispersion phenomenon of microbial abundance distribution

Starting from the individual level, different microbes' abundance distributions under different environmental conditions share some common probabilistic properties based on the counting data from 16S rRNA gene sequencing results (@fig-mean_var B). One of the most important properties is over-dispersion. For microbes in both natural (@fig-mean_var C) and experimental (@fig-mean_var D) environments, the variance ($\boldsymbol{V}$) and mean ($\boldsymbol{E}$) of microbial abundance are generally following a quadratic relationship as $$V_i=E_i+\frac{1}{k}E_i^2+\epsilon_i$$ {#eq-mean_variance} where $i$ denotes the index of microbe and $\epsilon_i$ denotes the random Gaussian noise such that $\epsilon_i \sim \mathcal{N}(0,\sigma_e^2)$. Notice that $V_i$ is greater than $E_i$, which is known as the dispersion pattern. Beyond that, @eq-mean_variance suggests that $V_i$ is expected to be much greater than $E_i$ when $E_i$ is large itself and $k$ is at a considerable scale. This phenomenon is known as the over-dispersion pattern. One may ask why we are intrigured by @eq-mean_variance. A quick but non-trivial answer is that the variance $\boldsymbol{V}$ in @eq-mean_variance has been serving as the most crucial brick of quantitative analysis in the field of ecology. Variance analysis lays the foundation of (dis)similarity, correlation, and even diversity evaluations. Following @eq-mean_variance, to appropriately characterize $\boldsymbol{V}$ for microbial abundance data, we need a model that can appropriately measure both $\boldsymbol{E}$ and $k$. We address the estimate of $\boldsymbol{E}$ as $\boldsymbol{\mu}$, which quantifies the microbial abundance at the population level. The other parameter $k$ quantifies the scale of the over-dispersion, i.e., how huge the gap is between $\boldsymbol{V}$ and $\boldsymbol{E}$.

```{r overdispersion_test, eval=FALSE, echo=TRUE, warning=FALSE, message=FALSE}
# load our package PMCosm
sapply(list.files(pattern="[.]R$", path="../R/PMCosm", full.names=TRUE), source)

# for AO data
mic <- read.table('../data/AO/OTU_table.txt')
id_dye <- c('LS','SF','CZ','BA','YF')
id_med <- c('GB','YTSW','XHC','ZC')
id_pes <- c('YT','YN')
lis_ind <- list(id_dye, id_med, id_pes)
names(lis_ind) <- c('Dye','Medicine','Pesticide')
id_inf <- c('Inf1','Inf2','Inf3')
id_axi <- c('Ax1','Ax2','Ax3')
id_oxi <- c('Ox1','Ox2','Ox3')
id_eff <- c('Eff1','Eff2','Eff3')
lis_pro <- list(id_inf, id_axi, id_oxi, id_eff)
names(lis_pro) <- c('Influent','Anoxic','Oxic','Effluent')

for(region in 1:4){
  # extract column id
  id_col <- c()
  for(i in 1:length(lis_ind)){
    id_ind <- lis_ind[[i]]
    id_pro <- lis_pro[[region]]
    id_col <- c(id_col, paste0(rep(id_ind, each=length(id_pro)),'_',rep(id_pro, length(id_ind))))
  }
  # extract abundance data
  mic_sub <- mic[,id_col]
  mic_sub <- mic_sub[rowMeans(mic_sub)>0,]
  
  # test zero-inflation
  id_zf <- apply(mic_sub, 1, is_zero_infla)
  # test over-dispersion
  id_od <- apply(mic_sub, 1, is_overdispersion)
  
  # report over-dispersion proportions among non zero-inflated
  print(paste0("the proportion of over-dispersion among non zero-inflated microbes in ", names(lis_pro)[region]," is: ",sum((!id_zf) & id_od)/sum(!id_zf)))
}


# for HZ data
mic <- read.table(file='../data/2018/resample_feature-table.txt',
                  header=TRUE,row.names=1)
# construct column id
bay_loci <- c('HB1.1','HB1.2','HB1.3','HB2.1','HB2.2','HB2.3',
              'HB3.1','HB3.2','HB3.3','HB4.1','HB4.2','HB4.3',
              'HB5.1','HB5.2','HB5.3','HB6.1','HB6.2','HB6.3',
              'HB7.1','HB7.2','HB7.3','HB8.1','HB8.2','HB8.3',
              'HB9.1','HB9.2','HB9.3','HB10.1','HB10.2','HB10.3')
era_sy_loci <- c('SY1.1','SY1.2','SY1.3','SY2.1','SY2.2','SY2.3',
                 'SY3.1','SY3.2','SY3.3','SY4.1','SY4.2','SY4.3',
                 'SY5.1','SY5.2','SY5.3','SY6.1','SY6.2','SY6.3')
era_jx_loci <- c('JX1.1','JX1.2','JX1.3','JX2.1','JX2.2','JX2.3',
                 'JX3.1','JX3.2','JX3.3','JX4.1','JX4.2','JX4.3',
                 'JX5.1','JX5.2','JX5.3','JX6.1','JX6.2','JX6.3')
mic_bay <- mic[,bay_loci]
mic_bay <- mic_bay[rowMeans(mic_bay)>0,]
mic_era <- mic[,c(era_sy_loci, era_jx_loci)]
mic_era <- mic_era[rowMeans(mic_era)>0,]
# test zero-inflation
id_zf_bay <- apply(mic_bay, 1, is_zero_infla)
id_zf_era <- apply(mic_era, 1, is_zero_infla)
# test over-dispersion
id_od_bay <- apply(mic_bay, 1, is_overdispersion)
id_od_era <- apply(mic_era, 1, is_overdispersion)
print(paste0("the proportion of over-dispersion among non zero-inflated microbes in Bay is: ", sum((!id_zf_bay) & id_od_bay)/sum(!id_zf_bay)))
print(paste0("the proportion of over-dispersion among non zero-inflated microbes in ERAs is: ", sum((!id_zf_era) & id_od_era)/sum(!id_zf_era)))
```

To confirm the existence of over-dispersion accurately, we conducted over-dispersion test for each microbe in both AO and Hangzhou data. Among microbes that are not zero-inflated in AO data, the proportions of over-dispersion ($p$-value $< 0.05$) are $83.33\%$ in *Influent* procedure, $96.83\%$ in *Anoxic* procedure, $98.70\%$ in *Oxic* procedure, and $88.71\%$ in *Effluent* procedure. For Hangzhou data, the proportions are $82.72\%$ in Bay and $85.04\%$ in ERAs.

```{r mean_var_pre, eval=FALSE, echo=FALSE, warning=FALSE, message=FALSE}
# example code for figure 1d
mic <- read.table('../data/AO/OTU_table.txt')

id_dye <- c('LS','SF','CZ','BA','YF')
id_med <- c('GB','YTSW','XHC','ZC')
id_pes <- c('YT','YN')
lis_ind <- list(id_dye, id_med, id_pes)
names(lis_ind) <- c('Dye','Medicine','Pesticide')

id_inf <- c('Inf1','Inf2','Inf3')
id_axi <- c('Ax1','Ax2','Ax3')
id_oxi <- c('Ox1','Ox2','Ox3')
id_eff <- c('Eff1','Eff2','Eff3')
lis_pro <- list(id_inf, id_axi, id_oxi, id_eff)
names(lis_pro) <- c('Influent','Anoxic','Oxic','Effluent')


dat_plt <- data.frame()
for(i in 1:length(lis_ind)){
  for(j in 1:length(lis_pro)){
    id_ind <- lis_ind[[i]]
    id_pro <- lis_pro[[j]]
    id_col <- paste0(rep(id_ind, each=length(id_pro)),'_',rep(id_pro, length(id_ind)))
    
    # extract microbe abundance data
    mic_sub <- mic[,id_col]
    mic_plt <- data.frame(miseqmean = rowMeans(as.matrix(mic_sub)),
                          miseqvar = rowVars(as.matrix(mic_sub)))
    mic_plt <- mic_plt[mic_plt$miseqmean>0,]
    
    mic_plt$Industry <- names(lis_ind)[i]
    mic_plt$Process  <- names(lis_pro)[j]
    
    dat_plt <- rbind(dat_plt, mic_plt)
  }
}

save(dat_plt, file='../data/AO/OTU_plot.RData')

load('../data/AO/OTU_plot.RData')
dat_plt$Industry <- factor(dat_plt$Industry, levels=c('Dye','Medicine','Pesticide'))
dat_plt$Process <- factor(dat_plt$Process, levels=c('Influent','Anoxic','Oxic','Effluent'))
ggplot(dat_plt, aes(x=miseqmean, y=miseqvar)) + 
    geom_point(alpha=0.36, size=1.5) +
    xlab(TeX('Abundance Mean ($E$)')) +
    ylab(TeX('Abundance Variance ($V$)')) +
    xlim(0,3000) +
    ylim(0,3e7) +
    theme_bw() +
    theme(panel.grid.major = element_blank(),
          panel.grid.minor = element_blank(),
          axis.title.x = element_text(size = 10),
          axis.title.y = element_text(size = 10),
          axis.text.x = element_text(size = 7),
          axis.text.y = element_text(size = 7)) +
    theme(aspect.ratio=1) +
    facet_grid(Industry~Process)
```

::: {#fig-mean_var}
![](../vignettes/figures/fig1.png)

**The over-dispersion phenomenon of microbial abundance across sample sites in natural and experimental environments. (A)** The geographical location of ten sample sites in Hangzhou Bay and two nearby ERAs. We refer to sites in Hangzhou Bay as HB1-10, sites in Jiaxing ERA as JX1-6, and sites in Shangyu ERA as SY1-6. We collected three replicated samples from each site. **(B)** The abundance distribution of three sulfate-reducing bacteria. Each type of bacteria has a site-specific abundance distribution while these distributions share a similar shape of their density curves. **(C)** The mean-variance relationship regarding the microbes' abundance is following a quadratic function in both Hangzhou Bay and ERAs. **(D)** The mean-variance relationship regarding the microbes' abundance is following a quadratic function in the AO system.
:::

```{r}
# run code
# ../vignettes/figures/ao_var_mean.Rmd
```

### The *Two-Wing* pattern of microbial communities

To fully retain the over-dispersion characteristic, a good candidate for quantifying the counting data is the Gamma-Poisson model (GPM). For each specific type of microbes, saying microbe $i$, define $x_{ij}$ as the minimum number or abundance of microbes that ensure the survival of microbe $i$ in sample site $j$. In 16S rRNA gene sequencing data, $x_{ij}$ represents the abundance of microbe $i$ under condition $j$. GPM quantifies $x_{ij}$ with a two-layer model as $$x_{ij}|\lambda_{ij}\sim \text{Poisson}(\lambda_{ij}),\,\,\,\,\,\,\lambda_{ij}\sim \text{Gamma}(\alpha,\beta).$$ {#eq-gpm} Then one can calculate the mean and variance as $$\mathbb{E}[x_{ij}]=\alpha \beta,\,\,\,\,\,\,\mathbb{V}(x_{ij})=\alpha \beta + \alpha^2\beta,$$ which satisfy the over-dispersion pattern in @eq-mean_variance. An equivalent way to express @eq-gpm is using the negative binomial distribution whereas $$x_{ij}|r_{ij},\varrho_{ij}\sim \text{NBin}(r_{ij},\varrho_{ij})\,\,\,\,\,\,\text{s.t.}\,\,\,\,\,\,\mathbb{P}(x_{ij}=x|r_{ij},\varrho_{ij})=\genfrac(){0pt}{2}{r_{ij}+x-1}{x}\varrho_{ij}^{r_{ij}}(1-\varrho_{ij})^{x}.$$ {#eq-nb} Now the mean and variance are calculated as $$\mathbb{E}[x_{ij}|r_{ij},\varrho_{ij}]=\frac{r_{ij}(1-\varrho_{ij})}{\varrho_{ij}},\,\,\,\,\,\,\mathbb{V}(x_{ij}|r_{ij},\varrho_{ij})=\frac{r_{ij}(1-\varrho_{ij})}{\varrho_{ij}^2},$$ which also satisfy the over-dispersion pattern in @eq-mean_variance. The connection between @eq-gpm and @eq-nb is $$r_{ij}=\beta,\,\,\,\,\,\,\varrho_{ij}=\frac{1}{\alpha+1}.$$ Comparing @eq-mean_variance and @eq-gpm, we notice that $$\mu=\alpha \beta,\,\,\,\,\,\,k=\frac{1}{\beta}.$$ Accurate estimations of the parameter pair $\mu$ and $k$ can appropriately quantify the over-dispersion pattern.

Specifically for the counting results extracted from 16S rRNA gene sequencing data, one may frequently encounter an inflated proportion of zeros. For microbes with this characteristic, we modify GPM in @eq-gpm as a zero-inflated Gamma-Poisson model (zGPM). We use an additional parameter $\pi_0$ to quantify the effect of unexpected zeros on estimating $\mu$ and $k$.

Altogether, we have a parameter trio $(\mu,k,\pi_0)$ to model the microbial abundance data. We calculated maximum-likelihood estimates (MLE) of the parameter trio for microbes in both the AO system and the Hangzhou Bay with two nearby effluent-receiving areas (ERAs). We discovered a *Two-Wing* pattern of microbial communities in both experimental and natural environments (@fig-two_wing).

::: {#fig-two_wing}
![](../vignettes/figures/fig2.png)

**The *Two-Wing* pattern of microbial communities in (A)** AO data and **(B)** Hangzhou Bay and two nearby ERAs. The gray dotdashed line denotes $\log(k)=7$. We address the left side of the gray dotdashed line as $\mu$-wing since it contains the highest estimate(s) of $\mu$. We address the right side of the gray dotdashed line as $k$-wing since it covers the highest estimate(s) of $k$.
:::

```{r}
# run code
# ../vignettes/figures/mu_k_ao.Rmd
# ../vignettes/figures/mu_k_hz.Rmd
```

### Microbes in $\mu$-wing are relatively minor but crutial for characterizing the community structure

Around $20\%$ species are enriched in $\mu$-wing in AO data across all four stages ($21.25\%$ in *Influent*, $21.65\%$ in *Anoxic*, $20.46\%$ in *Oxic*, $23.40\%$ in *Effluent*). In natural environments, $16.59\%$ species in the Hangzhou Bay are enriched in $\mu$-wing and $14.93\%$ species in nearby ERAs are enriched in $\mu$-wing. Microbes in $\mu$-wing has limited diversity compared to those in $k$-wing.

However, microbes in $\mu$-wing are sufficient enough to resemble the community structure of the whole cohort based on PCA and PCoA results (@fig-pca_pcoa). After simple rotations if necessary, PCA and PCoA results of $\mu$-wing are almost identical to PCA and PCoA results of the whole community. On the other hand, $k$-wing displays completely different PCA and PCoA results. No matter what rotations have been conducted, microbes in $k$-wing can not present the same or even close PCA and PCoA results of the whole community.

::: {#fig-pca_pcoa}
![](../vignettes/figures/fig3.png)

**PCA and PCoA results of all microbes, microbes in** $\mu$-wing, and microbes in $k$-wing. (A) PCA results of AO data. **(B)** PCA results of Hangzhou Bay and two nearby ERAs. **(C)** PCoA results of AO data. **(D)** PCoA results of Hangzhou Bay and two nearby ERAs. The abundance structure of microbes in $\mu$-wing resemble the structure of the whole microbial cohort. Microbes in $k$-wing display completely different structural pattern compared to the whole microbial cohort.
:::

```{r}
# run code
# ../vignettes/figures/pca_ao.Rmd
# ../vignettes/figures/pca_hz.Rmd
# ../vignettes/figures/pcoa_ao.Rmd
# ../vignettes/figures/pcoa_hz.Rmd
```

### Microbes in $\mu$-wing dominate canonical correspondence with environmental factors

In @fig-cca_ao, we conducted canonical correspondence analysis (CCA) for all microbes, microbes in $\mu$-wing, and microbes in $k$-wing for both AO data and Hangzhou Bay with nearby ERAs, as well as corresponding environmental factors. Similar to PCA and PCoA results, microbes in $\mu$-wing are sufficient enough to resemble the canonical correspondence of the whole community with environmental factors. Microbes in $k$-wing display completely different CCA results compared to the whole community.

::: {#fig-cca_ao}
![](../vignettes/figures/fig4.png)

**CCA results of all microbes, microbes in** $\mu$-wing, and microbes in $k$-wing in AO data.
:::

```{r}
# run code
# ../vignettes/figures/cca_ao_ori.Rmd
```

### Validating and extending existing rules to categorize microbial taxa

The *Two-Wing* pattern is a generalization of existing rules to categorize microbial taxa (@fig-wing_category). Recall that [@dai2016] has defined six microbial taxa categories as: (i) rare taxa (RT), OTUs with abundance $\leq 0.1\%$ in all samples; (ii) abundant taxa (AT), OTUs with abundance $\geq 1\%$ in all samples; (iii) moderate taxa (MT), OTUs with abundance between $0.1$ and $1\%$ in all samples; (iv) conditionally rare taxa (CRT), OTUs with abundance below $1\%$ in all samples and $\leq 0.1\%$ in some samples; (v) conditionally abundant taxa (CAT), OTUs with abundance greater than $0.1\%$ in all samples and $\geq 1\%$ in some samples but never rare (never $\leq 0.1\%$); and (vi) conditionally rare or abundant taxa (CRAT), OTUs with abundance varying from rare ($\leq 0.1\%$) to abundant ($\geq 1\%$). Here we follow many other researchers' suggestion [@chen2017; @mo2018; @xue2018; @liu2019] to use a threshold of $0.01\%$ instead of $0.1\%$ do define RT. We especially emphasize two important discoveries in the following subsections.

::: {#fig-wing_category}
![](../vignettes/figures/fig5.png)

**The categories of microbes in** $\mu$-wing and $k$-wing based on (A) the AO data and **(B)** the abundance data in Hangzhou Bay and nearby ERAs.
:::

```{r}
# run code
# ../vignettes/figures/mu_k_ao_category.Rmd
# ../vignettes/figures/mu_k_hz_category.Rmd
```

### The enrichment of non-rare taxa in $\mu$-wing

The four non-rare taxa categories--MT, AT, CAT, and CRAT--are all enriched in $\mu$-wing. These microbes have very small $k$ and thereby a very obvious over-dispersion characteristic.

::: {#fig-cca_hz_all_mu_nrt}
![](../vignettes/figures/cca_hz_all_mu_nrt_log.png)

**The CCA results of microbes in Hangzhou Bay and nearby ERAs.** The left column is the CCA result for the whole community. The middle column is the result for microbes in $\mu$-wing. The right column is the result for non-rare taxa.
:::

```{r}
# run code
# ../vignettes/figures/cca_hz_all_nrt_log.Rmd
```

### The sub-structures of RT and CRT

RT and CRT are the only two categories that exist in both $\mu$-wing and $k$-wing. This suggests that even among RT or among CRT, microbes may play different ecological roles. @koleff2003 has a good summary for interpreting $\beta$-diversity analysis results in @fig-rt_crt_mu_k G-K.

::: {#fig-rt_crt_mu_k}
![](../vignettes/figures/rt_crt_mu_k.png)

**RT and CRT in** $\mu$-wing and $k$-wing present different CCA and diversity evaluation results.
:::

```{r}
# run code
# ../vignettes/figures/diversity_ao_rt_crt_mu_k.Rmd
# ../vignettes/figures/diversity_hz_rt_crt_mu_k.Rmd
```

### Tracking functional microbes in $\mu$-wing across different environmental conditions

*Why do we consider microbes in* $\mu$-wing as potential functional candidates? Let us revisit the over-dispersion phenomenon that is quantified by @eq-mean_variance as $$V_i=E_i+\frac{1}{k}E_i^2+\epsilon_i.$$ In $\mu$-wing, the estimate of the scale parameter $k$ is relatively small. As a result, $V_i$ is considerably greater than $E_i$, which suggests a pattern of sever over-dispersion. For non-rare microbes in $\mu$-wing whose $E_i$ is large, their $V_i$ is at a very high level such that they can capture the heterogeneity of sample sites in the study area.

::: {#fig-mu_wing_mic}
![](../vignettes/figures/fig6.png)

**The locations of key microbes in the *Two-Wing* pattern through four stages of the AO system.**
:::

```{r}
# run code
# ../vignettes/figures/mu_k_ao_reducemic.Rmd
```

*Acinetobacter* (不动杆菌属) Usually a pathogen. Innately resistant to many antibiotics. Can spread rapidly among patients in ICU. Widely distributed in natural soil and water. Plays important role in the aerobic mineralization of organic matter [@isolatio].

*Bacillus* (芽孢杆菌属) Frequently used in industries due to its ability of producing useful enzymes. Species in this genus can be either obligate aerobes or facultative anaerobes.

*Caldisericum* (Caldisericum 属) Sulfur-reducing microbes. Capable of reducing sulfur compounds but not sulfate [@mori]. Has been reported in both natural and engineered habitats [@kim2014].

*Enterobacteriaceae* (肠杆菌科) Facultative anaerobes. Mostly ferment sugars. Some reduce nitrate to nitrite.

*Oligella* (寡养杆菌属) Mostly aerobic. Can cause infections. Most clinical isolates are from the urine.

*SM1A02* (SM1A02 属) Has been reported to be related to ammonia transformation in the activated sludge process [@tian2017].

*Wolinella* (沃林氏菌属) Most of them grow anaerobically by respiratory nitrite ammonification.

### Roles of the microbes in $k$-wing

For microbes in $k$-wing, the scale parameter $k$ is estimated to be considerably huge. Recall @eq-mean_variance whereas $$V_i=E_i+\frac{1}{k}E_i^2+\epsilon_i,$$ $V_i$ would be very close to $E_i$ for a huge scale parameter $k$, especially when $E_i$ is small. This approximate equality between $V_i$ and $E_i$ sets the microbial abundance distribution to be approaching to a Poisson distribution. One can interpret this phenomenon when assuming the microbes in $k$-wing have almost equal preference of the living environments across all sample sites. Each microbe has a chance to survive in an environmental unit which can be modeled by a Bernoulli experiment. If considering the sample site as a union of these environmental units, the sum of independent Bernoulli trials would yield a Binomial distribution. Then a later Poisson approximation leads to the approximate equality between $V_i$ and $E_i$, thereby validating what we have observed regarding the microbial abundance in $k$-wing.

## Discussions

-   We discovered a *Two-Wing* pattern of microbial communities in both natural environments and experimental settings. 

-   Our findings assist in understanding the admixture structure behind microbial communities.

-   The *Two-Wing* pattern provides a path to better generalize and further extend current taxa categorizing rules. It solves the debating regarding the threshold of defining rare taxa.

-   We utilized the *Two-Wing* pattern to elaborate the limitations of using popular dimensionality reduction methods to analyze the architecture of microbial communities and the abundance heterogeneity explained by environmental factors.

-   Recognizing the existence of the *Two-Wing* pattern leads to a potential strategy to detect functional microbes under arbitrary environmental disturbances.

-   The well-separated clusters in the *Two-Wing* pattern motivate the development of a probabilistic model suitable for microbial data with large community size and small/large number of observations.

-   The *Two-Wing* pattern has shown the power of @fisher1943's well-established model in analyzing microbial genomic data. 

-   ...

## Acknowledgements

This project is supported by `funding title`. I would like to express my sincere appreciations for our colleagues' contributions including but not limited to

-   Microbial and environmental data in Hangzhou Bay and nearby ERAs: Zhiguo Su, Yufan Zheng, ...
-   Microbial and environmental data in the AO system: Weidong Chen, ...
-   Drafting details about sample collection and the sequencing procedure: Zhiguo Su, ...

## Data and code availability

All our raw sequencing data are available in NCBI database under the accession number `#&%@##`. Analysis scripts for this study are available on GitHub at <https://github.com/YushiFT/two-wing-mic>.

## Web resources

...

## Additional session info

```{r}
sessionInfo()
```

## Fin

**Thanks for listening and reading!**

## References
